content/en/blog/_posts/2023-08-15-sidecar-kep.md

```diff
@@ -0,0 +1,113 @@
+
+- The CPU, memory, device, and topology manager are unaware of the sidecar container lifetime and additional resource usage, and will operate as if the Pod had lower resource requests than it actually does.
+- The output of `kubectl describe node` is incorrect when sidecars are in use. The output shows resource usage that's lower than the actual usage because it doesn't use the new resource usage calculation for sidecar containers.
+
+## We need your feedback!
+
+In the alpha stage, we want you to try out sidecar containers in your environments and open issues if you encounter bugs or friction points. We're especially interested in feedback about the following:
+The shutdown sequence, especially with multiple sidecars running
+The backoff timeout adjustment for crashing sidecars
+The behavior of Pod readiness and liveness probes when sidecars are running
+
+To open an issue, see the [Kubernetes GitHub repository](https://github.com/kubernetes/kubernetes/issues/new/choose).
+
+## What’s next?
+
+In addition to the known issues that will be resolved, we're working on adding termination ordering for sidecar and main containers. This will ensure that sidecar containers only terminate after the Pod's main containers have exited.
+
+We’re excited to see the sidecar feature come to Kubernetes and are interested in feedback.
+
+## Acknowledgements
+
+Many years have passed since the original KEP was written, so we apologize if we omit anyone who worked on this feature over the years. This is a best-effort attempt to recognize the people involved in this effort.
+
+- [mrunalp](https://github.com/mrunalp/) for design discussions and reviews
+- [thockin](https://github.com/thockin/) for API discussions and support thru years
+- [bobbypage](https://github.com/bobbypage) for reviews
+- [smarterclayton](https://github.com/smarterclayton) for detailed review and feedback
+- [howardjohn](https://github.com/howardjohn) for feedback over years and trying it early during implementation
+- [derekwaynecarr](https://github.com/derekwaynecarr) and [dchen1107](https://github.com/dchen1107) for leadership
+- [jpbetz](https://github.com/Jpbetz) for API and termination ordering designs as well as code reviews
+- [Joseph-Irving](https://github.com/Joseph-Irving) and [rata](https://github.com/rata) for the early iterations design and reviews years back
+- [swatisehgal](https://github.com/swatisehgal) and [ffromani](https://github.com/ffromani) for early feedback on resource managers impact
+- [alculquicondor](https://github.com/Alculquicondor) for feedback on addressing the version skew of the scheduler
+- [wojtek-t](https://github.com/Wojtek-t) for PRR review of a KEP
+- [ahg-g](https://github.com/ahg-g) for reviewing the scheduler portion of a KEP
+- [adisky](https://github.com/Adisky) for the Job completion issue
+
+## More Information
+
+- [Sidecar KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/753-sidecar-containers/README.md)
```

Comment:
Can we rebase the branch to add `API for sidecar containers` link that we previously included in https://github.com/kubernetes/website/pull/41802 somewhere?

Reply:
rebased

Reply:
```suggestion
- Read [API for sidecar containers](/docs/concepts/workloads/pods/init-containers/#api-for-sidecar-containers)
  in the Kubernetes documentation
- Read the [Sidecar KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/753-sidecar-containers/README.md)
```

Reply:
@matthyx 
Could you address this?